src/api/object_api.js

```diff
@@ -1156,6 +1156,9 @@ module.exports = {
                 properties: {
                     is_empty: {
                         type: 'boolean'
+                    },
+                    objects_deleted: {
```

Comment:
maybe call it `num_objects_deleted` ? 
```suggestion
                    num_objects_deleted: {
```